river/ensemble/adaptive_random_forest.py

```diff
@@ -207,15 +207,27 @@ def _new_leaf(self, initial_stats=None, parent=None):
 
         if self._leaf_prediction == self._MAJORITY_CLASS:
             return RandomLeafMajorityClass(
-                initial_stats, depth, self.splitter, self.max_features, seed,
+                initial_stats,
+                depth,
+                self.splitter,
+                self.max_features,
+                seed,
             )
         elif self._leaf_prediction == self._NAIVE_BAYES:
             return RandomLeafNaiveBayes(
-                initial_stats, depth, self.splitter, self.max_features, seed,
+                initial_stats,
```

Comment:
:thought_balloon: Why did you mix such a function parameter layout adjustment with changes for an other source code refactoring?

Reply:
It's just a change that got picked up by black.